### PR TITLE
Updated installation language files for 20 languages

### PR DESCRIPTION
**Pull request by Bakual**
To be merged when Travis passes. Can't directly commit those files anymore since staging is a protected branch.

### CODE REVIEW

File: installation/language/de-CH/de-CH.xml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-		<filename>de-CH.ini</filename>
-	</files>
-	<metadata>
-		<name>German (Switzerland)</name>
-		<nativeName>Deutsch (Schweiz)</nativeName>
-		<tag>de-CH</tag>
-		<rtl>0</rtl>
-	</metadata>
-	<params/>
+    <name>German (Switzerland)</name>
```
Comment (zero-24) [inline installation/language/de-CH/de-CH.xml]: is the change from tabs to spaces expected?

**Comment by zero-24** [comment]
Drone is happy now but we have a `tabs to space` change wich is IIRC not expected.

File: installation/language/de-CH/de-CH.xml (hunk 1 above)
Comment (Bakual) [inline installation/language/de-CH/de-CH.xml]: You know where you can fix it in the source 😄 
I'll fix it in my branch.

**Comment by zero-24** [comment]
Thanks @Bakual merged

**Pull request merged**